### PR TITLE
First android challenge

### PR DESCRIPTION
**Pull request by joninvski**
Add a new challenge for mobile android developers

### CODE REVIEW

**Comment by rfelix** [comment]
Looks like an interesting challenge!

File: problems/m0_call_tracker.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
+# Talkdesk Android Call Tracker
+
+At Talkdesk we want to offer our users the ability to track their outgoing calls and sms messages.
+
+Whenever a user makes an outgoing call or sends an sms, our app should register it and inform the talkdesk server of this occurrence.
+The record should include the time, the destination phone number, the current user location (does not need to be very precise) and the network type the user is on.
+Please note that the user can be offline (no internet connection) and still make calls or send sms.
+Probably you have to create a temporary local db.
```
Comment (ivoanjo) [inline problems/m0_call_tracker.md]: Can I suggest removing this hint and maybe rephrasing the previous sentence as 
> Please note that the user can be offline (no internet connection) and still make calls or send sms, and we would still want to be notified of those events when the device comes online.

File: problems/m0_call_tracker.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+
+Visually, the app should have a simple single screen UI where it can turn on/off the call and sms tracking.
+In the same screen it should be able to see how many calls/sms were tracked by Talkdesk.
+
+Assume that a user can have the app installed in several phones, and as so the local database does not provide the *real* value. You have to ask the server.
```
Comment (ivoanjo) [inline problems/m0_call_tracker.md]: I would rephrase as something like:
> Assume that a user can have the app installed in several phones, and these counters should be an aggregate of all the devices, rather than only the device's local counters.

File: problems/m0_call_tracker.md (hunk 1 above)
Comment (joninvski) [inline problems/m0_call_tracker.md]: I agree Ivo

File: problems/m0_call_tracker.md (hunk 2 above)
Comment (joninvski) [inline problems/m0_call_tracker.md]: Also agree

**Comment by ivoanjo** [comment]
Looks good :+1:
